apps/web/src/services/constants/milestone.constants.ts

```diff
@@ -3,4 +3,8 @@ export const milestoneRecruitmentOptions = [
   { value: '25', label: 'Interview Completed' },
   { value: '26', label: 'References Completed' },
   { value: '27', label: 'Offered Position' },
+  { value: '28', label: 'Candidate Withdrew' },
+  { value: '29', label: 'Candidate Was Unresponsive' },
+  { value: '30', label: 'Candidate Was Not Selected' },
+  { value: '31', label: 'Candidate Accepted the Job Offer' },
```

Comment:
Milestone values need to fetch from the database too. We can filter data for recruitment milestones and keep only required data in a database for it.

we have an API available for it. /api/v1/ienmaster/status
In this API you can fetch data for "id": 3, below the object.
{
"id": 3,
 "status": "IEN Recruitment (once licensed)"
 "children" : [ <array of an available milestone for the recruitment>]
}

If we push this data on Dev, in that database you will see different values for these milestones as these are auto-generated IDs.
if that is some big change here, we will plan in next PR. Let me know your thoughts.